include/itemlistformaction.h

```diff
@@ -92,6 +92,13 @@ class ItemListFormAction : public ListFormAction {
 		const std::string& url,
 		FmtStrFormatter& fmt);
 
+	std::vector<ItemPtrPosPair> visible_items;
+	int old_itempos;
+	std::shared_ptr<RssFeed> feed;
+	std::string search_phrase;
```

Comment:
Does `ItemListFormAction` really need this field? Can't it be moved to `SearchResultsListFormAction` too?

Reply:
@Minoru `search_phrase` used here https://github.com/newsboat/newsboat/blob/39f5e092fcef8cca9a010a57d5cad2866503f001/src/itemlistformaction.cpp#L1505, but I chek this `if` via `LOG` it doesn't seem to be go inside. Could we delete them?

Reply:
It can't be removed. This method is called when listing dialogs (press <kbd>v</kbd> to see the list). Without this `if`, `SearchResultsListFormAction` has a title of "Article List - ", which is weird.

I think the code that used to be inside this `if` should be moved into `SearchResultsListFormAction::title()`.